Polygon_mesh_processing/examples/Polygon_mesh_processing/locate_example.cpp

```diff
@@ -0,0 +1,113 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/AABB_face_graph_triangle_primitive.h>
+#include <CGAL/AABB_tree.h>
+#include <CGAL/AABB_traits.h>
+#include <CGAL/boost/graph/helpers.h>
+#include <CGAL/Dynamic_property_map.h>
+#include <CGAL/Polygon_mesh_processing/locate.h>
+#include <CGAL/Polygon_mesh_processing/triangulate_faces.h>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel             K;
+typedef K::FT                                                           FT;
+typedef K::Point_2                                                      Point_2;
+typedef K::Ray_2                                                        Ray_2;
+typedef K::Point_3                                                      Point_3;
+typedef K::Ray_3                                                        Ray_3;
+
+typedef CGAL::Surface_mesh<Point_3>                                     Mesh;
+typedef typename boost::graph_traits<Mesh>::vertex_descriptor           vertex_descriptor;
+typedef typename boost::graph_traits<Mesh>::face_descriptor             face_descriptor;
+
+namespace CP = CGAL::parameters;
```

Comment:
I don't mind too much, but it seems to me that everywhere else (at least in this package) we call this namespace `NP` for "Named Parameters".
Is there a reason to change that?